docs/pages/extensions/team-management/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Team Management in Meshery Cloud
+permalink: extensions/team-management
+language: en
+abstract: Learn how to efficiently add team members to Meshery Cloud, assign roles and permissions, and facilitate collaboration within the platform.
+display-title: "false"
+list: include
+type: extensions
+---
+
+## Seamlessly Add and Manage Team Members in Meshery Cloud
+
+ Meshery Cloud has the ability to efficiently add and manage team members within the platform's intuitive interface. Follow this comprehensive guide to seamlessly integrate new members and streamline collaboration within your Meshery Cloud team.
+
+* Log in to your Meshery Cloud account and access your cloud Dashboard.
+
+* In the Dashboard, navigate to your profile section or settings.
+
+* Look for the "Identity" section within your profile settings.
+
+* Under the "Identity" section, you will find options to create a new team or join an existing team.
+
+* To Invite a user, click on the "Invite User" or "Add User" button.
+
+* Enter the name and email and click on "Send Invite" button .
+
+* The team admin or owner will receive your request and review it. Once approved, you will become a member of the team.
+
+* If you want to create a new team, click on the "Create Team" button or a similar option.
+
+* Provide a suitable name for your team and any other required information.
+
+* After creating the team, you become the admin or owner of the newly created team.
+
+* As the team admin or owner, you can now invite other team members by following the steps outlined earlier in the conversation.
+
+By following these steps, you can successfully join an existing team or create a new team within Meshery Cloud, allowing you to collaborate and contribute to the team's service mesh management activities.
+
+## Notifying Team Members about Updates
+
+In Meshery Cloud, team members are notified about team updates through various communication channels to ensure they stay informed and engaged. Notifications are used to disseminate important information and keep team members up to date with the latest developments.
```

Comment:
Looking good, @yash37158. It's ideal to note Meshery Cloud's ability to integrate with third-party OAuth providers, too: GitHub and Google.

Reply:
sure @leecalcote, I'll take that into consideration. also, I was looking forward to improving the notifications section, as I couldn't find resources on that topic. can you please help me with that too?

Reply:
@yash37158   , the team invites generally goes through the email.